tests/manage/pv_services/test_daemon_kill_during_pvc_pod_deletion_and_io.py

```diff
@@ -120,6 +129,7 @@ def setup_base(self, interface, multi_pvc_factory, pod_factory):
                     interface=interface,
                     pvc=pvc_obj,
                     status="",
+                    node_name=next(nodes_iter),
```

Comment:
done we need to check `nodes_iter`, the same as in other places? (`next(nodes_iter) if nodes_iter else None`)

Reply:
In this case the list *nodes* will not be empty.
So I will update 
```nodes_iter = cycle(nodes) if nodes else None``` 
to ```nodes_iter = cycle(nodes)```

Reply:
Fixed